test/unit/url.cpp

```diff
@@ -1828,6 +1939,7 @@ class url_test
         testSegments();
         testResolution();
         testOstream();
+        testNormalize();
```

Comment:
Tests look nice and light, how is coverage?

Reply:
Almost 100% covered.